survey-sdk/src/main/java/co/nimblehq/survey/sdk/entity/AnswerEntity.kt

```diff
@@ -0,0 +1,32 @@
+package co.nimblehq.survey.sdk.entity
+
+import com.squareup.moshi.Json
+import moe.banana.jsonapi2.JsonApi
+
+@JsonApi(type = "answer")
+class AnswerEntity : QuestionEntity() {
```

Comment:
What do you think about using `data class` for Entity classes?

Reply:
@hoangnguyen92dn Not in this case, as the `JsonApi` required empty default constructor for each class. Otherwise, it will throw an exception on runtime.

Reply:
Can you show me the exception? 🤔  Just one more thing, any reason why do we need the `AnswerEntity` extend from `QuestionEntity`? If the purpose is for reusing the same properties from `QuestionEntity` then I don't think so. These entities should be different from each other.

Reply:
@hoangnguyen92dn I'm not sure why but the exception was gone with the current implementation. So I will change them to `data class`.
For the purpose of `AnswerEntity`,  yes, just reusing the properties from `QuestionEntity`.
Ok, let remove the inheritance between them.